### ISSUE TITLE
Proposal: default root route

### ISSUE THREAD

**Issue by raghubetina** [firstdraft/appdev_template#143]
Proposal:

What about adding a default root route to all projects?

```
get("/", { :controller => "rails/info", :action => "routes" })
```

Thoughts @pmckernin @jelaniwoods ?

**Comment by pmckernin** [comment]
What is the reasoning?

**Comment by raghubetina** [comment]
@pmckernin I figured you might not be familiar with that so the main reason for this issue is to give you a chance to investigate if not.

Try to come up with what you think my reasoning might be.

**Comment by jelaniwoods** [comment]
@raghubetina I like this idea. I think this would be a good addition with the exception of maybe the first HTML projects that we do before any routing stuff. For those, I think the "Yay! You're on Rails" page is better since it looks less intimidating.

**Comment by raghubetina** [comment]
@jelaniwoods Good point. Perhaps `template.rb` should include an `ask` to find out whether to use this root route or not.
